portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,43 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+/* Servlet that returns json of commentaries statistics*/
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
+  /* Some hardcoded examples*/
+  String [] commentList = {"Hi! I'm Mark", "Hello, my name is Nancy :3", "Good morning/afternoon/evening! I'm Alex"};
+  String [] nameList = {"Mark", "Nancy", "Alex"};
+  String [] ratingList = {"4/5", "5/5", "0/5"};
```

Comment:
If ratings are always a number between 0 and 5, it should be an enum or integer. This can then be converted to '4/5` when displaying it in the HTML page, but it allows you to change how it is diplayed in future (e.g. show is as a number of stars, or a pie chart).